kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -436,7 +412,12 @@ def retrieve(
             )
 
     def _retrieve_filename(self, id_: str) -> Tuple[str, str]:
-        response = requests.get(id_, headers=self.service.headers, timeout=REQUEST_TIMEOUT)
+        response = requests.request(
+            method=hdrs.METH_GET,
```

Comment:
why did you change the requests to the general method here? and keep `requests.get` in other places

Reply:
Uh, I went for a find and replace streak, and I may have done it in places that weren't necessary, I can undo that one

Reply:
hmm maybe make them all consistent? I don't know ...

Reply:
They were definitely motivated by something for some methods, and not for others. I could keep the ones with valid motivation as request.request and revert back to the specialised method for the others

Reply:
I've undone the ones that weren't necessary